app/controllers/users_controller.rb

```diff
@@ -61,11 +62,11 @@ def handle_invitation
   end
 
   def payment_processor
```

Comment:
In my opinion, this method name does not reflect what the method does. I was confused when I read line 14 above.